### PR TITLE
Support multiple !Included schemas.

### PR DESCRIPTION
**Pull request by limbo**

### CODE REVIEW

**Comment by cybertk** [comment]
I fixed CI errors on master branch, could you please rebase your patches against master?

**Comment by limbo** [comment]
Made some additional changes and squashed all commits.

File: test/unit/test-test.coffee [hunk 1]
```diff
@@ -43,10 +43,7 @@ describe 'Test', ->
         test.request.body =
           body: 'value'
         test.response.status = 201
-        test.response.schema = """
-          type: 'string'
-          name: 'string'
-        """
+        test.response.schema = [{ type: 'object', properties: { type: 'string', name: 'string'}}]
```
Comment (cybertk) [inline test/unit/test-test.coffee]: I prefer csonschema for cleaner code

File: src/test.coffee [hunk 2]
```diff
@@ -79,7 +69,7 @@ class Test
 
     # Body
     if @response.schema
-      schema = @parseSchema @response.schema
+      schema = @response.schema
```
Comment (cybertk) [inline src/test.coffee]: I think we should keep `test.schema` and `test.body` in same type. Now, `test.body` is dict and `test.schema` is a string. Do you have any better ideas?

File: test/unit/test-test.coffee (hunk 1 above)
Comment (limbo) [inline test/unit/test-test.coffee]: No particular reason except that this is what we're using so it's what I'm used to.

File: src/test.coffee (hunk 2 above)
Comment (limbo) [inline src/test.coffee]: Actually, I changed schema to be an array of objects (parsed schemas). See my note on add-tests.coffee. Should probably have changed the field's name to schemas (plural) to make that clearer.

File: src/add-tests.coffee [hunk 3]
```diff
@@ -68,8 +78,12 @@ addTests = (raml, tests, parent, callback) ->
 
         # Update test.response
         test.response.status = status
-        test.response.schema = res?.body?['application/json']?.schema
-
+        test.response.schema = []
```
Comment (limbo) [inline src/add-tests.coffee]: Here is the change to make schema into an array of parsed schemas.

**Comment by cybertk** [comment]
Thanks for the contribution. As this PR changes API of `test` object, it will break current usage flow except for hooks files. Could you try to keep compatibility while adding this new feature?

I prefer keep `test.schema` as string type.

**Comment by cybertk** [comment]
Any update?

**Comment by limbo** [comment]
Sorry, got busy with other things here...

From what I can see, the schemas from the RAML parser are being passed as objects (parsed schemas as mentioned above). It made sense to make all schemas of that type so that we can have one big array with all the schemas in it. It seems to work fine for our use cases here and the tests are still passing so I think it's an OK solution. However, it's your project and your call.

**Comment by cybertk** [comment]
It's perfect solution. Could you help implement in this way?

**Pull request merged**